import/permissions/permissionsplugin.cpp

```diff
@@ -0,0 +1,45 @@
+/****************************************************************************
```

Comment:
No need to introduce new import for this one. At least, this is getting too detailed. We intentionally used broad import for text selection called "Controls". I think that this can be seen a control as well.